be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MessageDataContentValidator.scala

```diff
@@ -2,11 +2,14 @@ package ch.epfl.pop.pubsub.graph.validators
 
 import akka.pattern.AskableActorRef
 import ch.epfl.pop.model.network.JsonRpcRequest
-import ch.epfl.pop.model.objects.{Hash, PublicKey, Timestamp, WitnessSignaturePair}
+import ch.epfl.pop.model.objects.{Base64Data, Channel, Hash, PublicKey, Timestamp, WitnessSignaturePair}
```

Comment:
You can remove the `Base64Data` import now :)

Reply:
Some day my PRs will have proper imports first try, fixed !